src/cli/pm_trusted_command.zig

```diff
@@ -349,7 +349,7 @@ pub const TrustCommand = struct {
                     }
 
                     switch (pm.options.log_level) {
-                        inline else => |log_level| try pm.spawnPackageLifecycleScripts(ctx, info.scripts_list, log_level),
+                        inline else => |log_level| try pm.spawnPackageLifecycleScripts(ctx, info.scripts_list, log_level, false),
```

Comment:
Can you add a `/* what this param is */ false)` to the above. This hits on the boolean trap: https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Reply:
Updated. Assigned to variable instead of comment